### DISCUSSION TITLE
Welcome to hunajaholisti-homepage Discussions!

### DISCUSSION THREAD

**Discussion by nikohoffren** [nikohoffren/hunajaholisti-website#30] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by vasfvitor** [comment]
I have a suggestion: To add feedback when the user touches "add to cart" to show that was added to the cart, not only animate the button. Right now, visually, nothing happens on mobile.

**Reply by vasfvitor** [reply]
another that should be interesting, I guess, is to make all the cards in the same row be the equal in height, and keep the CTA aligned at the bottom.

**Reply by nikohoffren** [reply]
Good ideas, especially the feedback when the user adds items to the cart. At the moment there is only a underline that appears when the "add to cart" is clicked but some kind of modal or other text could be nice. I think i add both of those to the issues later on.
